extensions/wikia/AdEngine/js/slot/inContent.js

```diff
@@ -0,0 +1,58 @@
+/*global define*/
+define('ext.wikia.adEngine.slot.inContent', [
+	'ext.wikia.adEngine.adTracker',
+	'wikia.log',
+	'wikia.window'
+], function (adTracker, log, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.slot.inContent',
+		selector = '#mw-content-text > h2',
+		header;
+
+	/**
+	 * Adds dynamically new slot in the right place and sends tracking data
+	 */
+	function init(slotName) {
+		var adHtml = '<div id="' + slotName + '" class="wikia-ad default-height"></div>',
+			logMessage,
+			logWikiData = '(wikiId: ' + win.wgCityId + ' articleId: ' + win.wgArticleId + ')',
+			$header,
+			slotNameGA;
+
+		if (!slotName) {
+			log('slotName is falsy', 'error', logGroup);
+			return;
+		}
+
+		slotNameGA = slotName.toLowerCase();
+
```

Comment:
I'm thinking of tracking inits of in-content slots here. Something like:
```
adTracker.track('slot/' + slotNameGA + '/init')
```
What do you think about the idea (cc: @fraszczakszymon, @jacekjursza, @nefretete)?